Enterprise-CMCS/macpro-mdct-mcr
# YoY Copyover fixes

### Description

- update list of reports to copy from to not include archived reports
- updated wording on dropdown component title, dashboard "add" button and modal title to say "add / copy..."
- updated dropdown to be disabled if there are no reports
- updated dropdown to be disabled in the edit program modal 
- disabled dropdown in edit modal now shows which report was copied (if one was copied)



### Related ticket(s)


MDCT-2827

---
### How to test

- create a new program
- if there are submitted programs that haven't been archived, they should show up in the `copy an existing report` dropdown
- if there is a report to choose, choose one
- create new program, once it appears in the dashboard list, click the "edit report" icon on the left
- no matter what, the `copy an existing report` dropdown should be disabled. If you choose to copy a report, it should display as the default value in the disabled dropdown

no submitted reports available to copy from:
<img width="490" alt="Screenshot 2023-08-24 at 2 21 25 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/10237149/2903ae70-162d-420f-a236-7bd3cec23aae">

non-copied program, in the edit modal view:
<img width="514" alt="Screenshot 2023-08-24 at 2 26 36 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/10237149/e100e9e4-7643-427e-9048-e6054ee1660f">

create a new program, now there are report options to copy from:
<img width="544" alt="Screenshot 2023-08-24 at 2 27 25 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/10237149/570ce0ff-42b3-4f43-991f-b5ee9b808c91">

copied program, in the edit modal view:
<img width="525" alt="Screenshot 2023-08-24 at 2 27 15 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/10237149/91da288a-b237-4e8b-939f-d16cf8fd289a">




### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
